deisctl/config/etcd.go

```diff
@@ -42,6 +42,14 @@ func (c *etcdClient) Get(key string) (string, error) {
 	return resp.Node.Value, nil
 }
 
+func (c *etcdClient) Delete(key string) error {
+	_, err := c.etcd.Delete(key, false)
+	if err != nil {
```

Comment:
we should remove all this `if` logic and just return err, since we return it in both cases (nil and non-nil)